### DISCUSSION TITLE
How to make custom title for Tags page ?

### DISCUSSION THREAD

**Discussion by mvrck96** [adityatelange/hugo-PaperMod#619] [Q&A] [answered]
Hello people ! 

I'm setting up my site in home-info mode and run into problem. What is the proper way to make custom title to Tags page ?

In demo version search and archive pages can be renamed easily by changing title in `search.md` and `archive.md`. But when I create `tags.md` and change title there, title of the page ins't rendering properly. 

To recreate my issue you can:
- create `tags.md` in `content` directory
- enable tags button in `config.yml` under `menu --> main` section
- paste template from `archive.md` to `tags.md`
- rename `title` variable and other vars in `tags.md`
- render !

I'm running latest PaperMod and Hugo v0.89.0, OS: ubuntu 18.04

**Comment by danielfdickinson** [comment, marked as answer]
I believe (based on https://gohugo.io/content-management/taxonomies/#add-custom-metadata-to-a-taxonomy-or-term) that your issue is that you need to use /content/tags/_index.md​​

You need the _index.md​​ because a taxonomy (like 'tags') page needs to be a 'list' page not a 'single' page.
[https://gohugo.io/images/gohugoio-card.png]<https://gohugo.io/content-management/taxonomies/#add-custom-metadata-to-a-taxonomy-or-term>
Taxonomies | Hugo<https://gohugo.io/content-management/taxonomies/#add-custom-metadata-to-a-taxonomy-or-term>
Default Destinations . When taxonomies are used—and taxonomy templates are provided—Hugo will automatically create both a page listing all the taxonomy’s terms and individual pages with lists of content associated with each term. For example, a categories taxonomy declared in your configuration and used in your content front matter will create the following pages:
gohugo.io
See also: https://gohugo.io/templates/lists#add-content-and-front-matter-to-list-pages
[https://gohugo.io/images/gohugoio-card.png]<https://gohugo.io/templates/lists#add-content-and-front-matter-to-list-pages>
Lists of Content in Hugo | Hugo<https://gohugo.io/templates/lists#add-content-and-front-matter-to-list-pages>
A list page template is a template used to render multiple pieces of content in a single HTML page. The exception to this rule is the homepage, which is still a list but has its own dedicated template.. Hugo uses the term list in its truest sense; i.e. a sequential arrangement of material, especially in alphabetical or numerical order. Hugo uses list templates on any output HTML page where ...
gohugo.io
HTH

________________________________
From: Ivan ***@***.***>
Sent: November 3, 2021 5:35 PM
To: adityatelange/hugo-PaperMod ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [adityatelange/hugo-PaperMod] How to make custom title for Tags page ? (Discussion #619)


Hello people !

I'm setting up my site in home-info mode and run into problem. What is the proper way to make custom title to Tags page ?

In demo version search and archive pages can be renamed easily by changing title in search.md and archive.md. But when I create tags.md and change title there, title of the page ins't rendering properly.

To recreate my issue you can:

  *   create tags.md in content directory
  *   enable tags button in config.yml under menu --> main section
  *   paste template from archive.md to tags.md
  *   rename title variable and other vars in tags.md
  *   render !

I'm running latest PaperMod and Hugo v0.89.0, OS: ubuntu 18.04

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fadityatelange%2Fhugo-PaperMod%2Fdiscussions%2F619&data=04%7C01%7C%7C3feb85f668774c42061308d99f11da3a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637715721286003401%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=HP2%2BpdZ6TeCMDNVP8d%2FHGKaq0VGy9Yx%2FZ4UtXSk1VyY%3D&reserved=0>, or unsubscribe<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAE6GOSU46G7ZTRH5A5ZGJHLUKGTJ7ANCNFSM5HJ7UXIA&data=04%7C01%7C%7C3feb85f668774c42061308d99f11da3a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637715721286013395%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=EWOX5Ltv8kPSNrFu1%2FHh5UpNWMcq5hIgda%2FDhDGzIIA%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C3feb85f668774c42061308d99f11da3a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637715721286023391%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=GHbM2wX%2F0RhUOhQGe6OLA8s12gFJ2oUOc%2B6xyNOwAKM%3D&reserved=0> or Android<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C3feb85f668774c42061308d99f11da3a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637715721286023391%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=cvqUaNb%2BBosHvFgSHL3ajjbXSAaO0Pc%2FOBfiSVNdVJU%3D&reserved=0>.

**Reply by mvrck96** [reply]
Thank you for youe broad reply ! Adding `/content/tags/_index,md` solved my problem.

I'm quite new to hugo framework. Looks like I need to dive into hugo docs to figure out hove taxonomies probably work))

Thanks again !
